secboot/secboot_tpm.go

```diff
@@ -21,11 +21,14 @@
 package secboot
 
 import (
```

Comment:
we need to split and/or rename this file at some point, is not really about just TPM anymore